rg.go

```diff
@@ -0,0 +1,61 @@
+package brdoc
+
+import (
+	"bytes"
+	"regexp"
+	"unicode"
+)
+
+var (
+	RGRegexp = regexp.MustCompile(`^\d{2}\.?\d{3}\.?\d{3}-?[0-9xX]$`)
+)
+
+func IsRG(doc string) bool {
```

Comment:
Já que RG é algo específico por estado, creio que deveríamos aceitar o `FederativeUnit`, semelhante ao `IsCEP`, o que acha? Neste caso, seria interessante fazer umas coisas extras:
- Separar `FederativeUnit` do arquivo `cep.go`
- Dar `panic` caso seja algo diferente de SP ou RJ (já que não temos implementado)
- Talvez testar esse `panic`
- Adicionar no comentário/doc da função sobre o `panic` e os estados que permitimos

Reply:
Opa, desculpe pela super demora do meu lado.
Será que ao invés do `panic` não faria sentido retornar um erro de "federative unit not implemented"?

Reply:
Fiz um commit exemplificando como ficaria, veja o que acha  ;)